dcp_prototype/backend/browser/code/common/authorizer.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+token_prefix = "Bearer "
+userinfo_endpoint = "https://czi-single-cell.auth0.com/userinfo"
+
+
+def assert_authorized(header: dict):
+    resp = requests.post(userinfo_endpoint, headers={"Authorization": header["Authorization"]})
```

Comment:
Just out of curiosity, how does using the the `/userinfo` endpoint differ from the authorization setup [described here](https://auth0.com/docs/quickstart/backend/python#protect-api-endpoints)?

Reply:
That setup is following Oauth2 standard and is only verifying the Access token is valid before proceeding. We are following the ODIC standard which uses the access token to verify the user's identity with the identity provider using /userinfo before proceeding. We don't have the Access token verification step because we are using the opaque token provided by auth0. We will switch to a JWT Access token in a following PR. Using the JWT Access token will allow us to embed scopes into the token in the future.

Reply:
Gotcha. Thank you for the explanation.